src/questions.js

```diff
@@ -0,0 +1,149 @@
+const inquirer = require("inquirer");
+const roleList = (employeeRoles) => {
```

Comment:
The `roleList` and `updateEmployeeChoice` functions should not be in this file as it should solely include the questions. They can be in a separate utils folder as these can be reused for several of the different actions.